example_chain/testutil/eth_setup.go

```diff
@@ -111,6 +111,9 @@ func EthSetupWithDB(isCheckTx bool, chainID string, patchGenesis func(*example_a
 // NewTestGenesisState generate genesis state with single validator
 //
 // It is also setting up the EVM parameters to use sensible defaults.
+//
+// TODO: are these different genesis functions necessary or can they all be refactored into one?
+// there's also other genesis state functions; some like app.DefaultGenesis() or others in test helpers only.
```

Comment:
@coderabbitai Please create a corresponding ticket to clean this up - we have multiple genesis state functions for different testing utilities; I think some of them can be refactored for simplicity